packages/engine/lib/execution/src/runner/python/setup.sh

```diff
@@ -73,5 +73,6 @@ fi
 echo "Running setup.py"
 # Also compile Cython. Can be done either after
 # or before compiling Rust.
+export GITHUB_OUTPUT_FILE_PATH=github_output.txt
```

Comment:
This is not related to GitHub but is only used for the engine. I doubt this is needed

Reply:
I see, you are right. I removed this line.